Gruntfile.js

```diff
@@ -194,7 +196,12 @@ module.exports = function(grunt) {
             server: {
                 options: {
                     port: 8000,
-                    base: "."
+                    base: {
+                        path: ".",
+                        options: {
+                            stylesheet: 'style.css'
```

Comment:
What's the purpose of this?

Reply:
It inserts a custom stylesheet for the DEV page. I re-styled folders in the server lists so that it is easier to navigate.

Reply:
![image](https://github.com/user-attachments/assets/69c01d20-9bfe-42a5-9d75-4742bf191c46)
I mean, every time I go to the original OSD repo and try to navigate in it I get immediately lost. I got used to this already :D

Reply:
Cool. Nice to have prettier styling.